docs/reference/nuget-config-file.md

```diff
@@ -347,12 +347,12 @@ Below is an example `nuget.config` file that illustrates a number of settings:
     -->
     <trustedSigners>
         <author name="microsoft">
-		    <certificate fingerprint="3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE" hashAlgorithm="SHA256" allowUntrustedRoot="false" />
```

Comment:
This line still has a tab. See: https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#example-config-file